conjure_oxide/tests/integration/xyz/input.expected-minion.solutions.json

```diff
@@ -1,12 +1,12 @@
 [
   {
-    "a": 1,
-    "b": 1,
-    "c": 2
+    "UserName(a)": 1,
+    "UserName(b)": 1,
+    "UserName(c)": 2
   },
   {
-    "a": 2,
-    "b": 1,
-    "c": 1
+    "UserName(a)": 2,
```

Comment:
is this change on purpose? no problem if so.

Reply:
Minion / minion_rs stores names as strings always, but conjure AST has different types of names (Username, MachineName). 

Previously, the minion_rs AST types were used to get results, and the tester JSON used them for its output as well.
Now the tester  and its generated output uses conjure AST types. This means that the outputted names can be MachineNames or UserNames.


(as discussed in meeting)